lib/google_drive/collection.rb

```diff
@@ -60,6 +60,24 @@ def files(params = {}, &block)
       files_with_type(nil, params, &block)
     end
 
+    # Same as Session#upload_from_file. It uploads file to current collection
+    def upload_from_file(path, title = nil, params = {})
+      params[:parents] = [self.id]
```

Comment:
It's not good practice to modify an object (params) passed from the caller. Use this instead:

    params = {parents: [self.id]}.merge(params)

Same for below.